src/mesido/pycml/component_library/milp/gas/gas_base.py

```diff
@@ -15,7 +15,7 @@ def __init__(self, name, **modifiers):
         # TODO: think of more elegant approach for Q_shadow, currently required to ensure that
         #  every port has a unique variable to make the correct port mapping
         self.add_variable(Variable, "Q")  # [m3/s]
-        self.add_variable(Variable, "Q_shadow")
+        # self.add_variable(Variable, "Q_shadow")
```

Comment:
Can we remove Q_shadow completely or do you want to keep in the comments?

Reply:
should be removed now